api/src/main/java/io/opencensus/metrics/Metric.java

```diff
@@ -81,19 +81,16 @@ private static void checkTypeMatch(MetricDescriptor.Type type, List<TimeSeries>
           case GAUGE_INT64:
           case CUMULATIVE_INT64:
             Utils.checkArgument(
-                value instanceof ValueLong,
-                String.format("Type mismatch: %s, %s.", type, valueClassName));
+                value instanceof ValueLong, "Type mismatch: %s, %s.", type, valueClassName);
```

Comment:
Do you want to split this change to another commit? (`metrics` is not present in the v0.15 release)

Reply:
Done.